### ISSUE TITLE
SCN8A missing likely pathogenic ClinVar annotaiton

### ISSUE THREAD

**Issue by tonydisera** [tonydisera/gene.iobio#785]

**Comment by tonydisera** [comment]
Yes, I see the same issue.  Hmm..  We did switch over to using the ClinVar vcf rather than EUTILS to annotate the variants.  I will look into this.

**Comment by tonydisera** [comment]
The ClinVar annotation used to be in the ClinVar vcf and I was able to looking it up based on the ClinVar accession number:

https://www.ncbi.nlm.nih.gov/clinvar/RCV000189268.1/

There is a (tiny) ! not current glyph in the top right corner.
<img width="802" alt="screen shot 2018-01-29 at 8 53 58 am" src="https://user-images.githubusercontent.com/8420103/35519672-22776d38-04d2-11e8-8487-fd50d3b01a2f.png">

**Comment by tonydisera** [comment]
In ClinVar, the advanced search based on Chromosome and GRCh37 position returns zero results for this variant.  In dbSNP, the ClinVar entry is also omitted.
<img width="999" alt="dnsnp" src="https://user-images.githubusercontent.com/8420103/35520109-4e7cf19a-04d3-11e8-90d3-dd65304eb55f.png">

**Comment by AlistairNWard** [comment]
Ok, two comments:

1. Looking at sample ...976 for the same study. Gene SCN2A shows no ClinVar variants, but there is a variant at position 2:166165903 that does appear in a ClinVar search of the SCN2A gene.

2. Is there a way of showing ClinVar history? When I look at a sample that I know had a ClinVar variant and I no longer see it, it would be useful to see in the summary that a variant had been removed from ClinVar.
